### PR TITLE
New for 2020

### LINKED ISSUE #170: Add list of breaking changes for 2020
Examples include `TimedRobot::GetPeriod()` returning a `units::second_t` instead of `double`, etc.

### PR DESCRIPTION
**Pull request by Daltz333**
Revisions and comments welcome

Closes #170

### CODE REVIEW

File: source/docs/software/getting-started/new-for-2020.rst [hunk 1]
```diff
@@ -1,13 +1,81 @@
...
-- PIDController has been rewritten
-  - Takes lambdas for the measurements and outputs
-  - No longer runs asynchronously. Async has been a footgun for a large majority of teams, and a new implementation will be implemented in the future.
-- Kinematics classes have been added for Swerve, Mecanum, and DifferentialDrive. This allows for more complete closed loop driving of these drive types.
\ No newline at end of file
+The core WPILib library has received many changes. These include rewrites to major part of the core library, as well as a suite of kinematics to improve closed loop driving. The full change log can be read below.
+
+.. todo:: Add links to the specific usage guides for LinearDigitalFilter, and PIDController.
+
+- CommandBased has been rewritten from the ground up. The explanation on why can be found `here <https://github.com/Oblarg/docs/blob/master/CommandRewriteDesignDoc.md>`__.
```
Comment (AustinShalit) [inline source/docs/software/getting-started/new-for-2020.rst]: @Oblarg Can you open a PR to add this document to https://github.com/wpilibsuite/design-docs

Then we can update this link to point at the design docs repo.

Comment (Oblarg) [reply source/docs/software/getting-started/new-for-2020.rst]: Done.

File: source/docs/software/wpilib-overview/new-for-2020.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+- ScreenSteps has been replaced with `frc-docs <https://docs.wpilib.org/en/latest>`__. This change has largely been inspired by growing demands by the WPILib team. We hope that this move toward a community-led model will increase the quality of tutorials and documentation moving forward.
+
+WPILib
+------
+
+There are many changes and additions to the main WPILib library for 2020. Most notably, there is a new version of the command-based framework with several major enhancements, a new (synchronous) PIDController, and kinematics classes have been added for closed loop driving. The full change log can be read below.
+
+.. todo:: Add links to the specific usage guides for LinearDigitalFilter, and PIDController.
+
+- The command-based framework has been rewritten. The design rationale behind the rewrite can be found `here <https://github.com/Oblarg/docs/blob/master/CommandRewriteDesignDoc.md>`__. The new version of the framework is located in the ``frc2`` namespace (C++) and the ``edu.wpi.first.wpilibj2`` package (Java).
```
Comment (AustinShalit) [inline source/docs/software/wpilib-overview/new-for-2020.rst]: Can you add a TODO here to replace the link with the design docs one: https://github.com/wpilibsuite/design-docs/pull/7

**Pull request merged**